docs/resources/rf_stack.md

```diff
@@ -0,0 +1,194 @@
+    }
+  },
+  "resource": {
+    "huaweicloud_vpc": {
+      "test": {
+        "name": "$${var.vpc_name}",
+        "cidr": "192.168.0.0/16"
+      }
+    },
+    "huaweicloud_vpc_subnet": {
+      "test": {
+        "vpc_id": "$${huaweicloud_vpc.test.id}",
+        "name": "$${var.subnet_name}",
+        "cidr": "$${cidrsubnet(huaweicloud_vpc.test.cidr, 4, 1)}",
+        "gateway_ip": "$${cidrhost(cidrsubnet(huaweicloud_vpc.test.cidr, 4, 1), 1)}"
+      }
+    }
+  },
+  "variable": {
+    "region_name": {
+      "type": "string"
+    },
+    "vpc_name": {
+      "type": "string"
+    },
+    "subnet_name": {
+      "type": "string"
+    }
+  }
+}
+EOT
+
+  vars_body = <<EOT
+region_name = "${var.region_name}"
+vpc_name    = "${var.vpc_name}"
+subnet_name = "${var.subnet_name}"
+EOT
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region where the RF resource stack is located.  
+  If omitted, the provider-level region will be used. Change this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the resource stack.  
+  The name can contain `1` to `64` characters, only letters, digits and hyphens (-) are allowed.
+  The name must start with a lowercase letter and end with a lowercase letter or digit.
+  Change this parameter will create a new resource.
+
+* `description` - (Optional, String, ForceNew) Specifies the description of the resource stack, which contain maximum of
+  255 characters.  
+  Change this parameter will create a new resource.
+
+* `agency` - (Optional, String, ForceNew) Specifies the configuration of the agencies authorized to IAC.  
+  Change this parameter will create a new resource.
+  The [object](#stack_agency) structure is documented below.
+
+* `template_body` - (Optional, String) Specifies the HCL/JSON template content for deployment resources.
+  This parameter and `template_uri` are alternative and required if `vars_body` is set.
+
+* `vars_body` - (Optional, String) Specifies the variable content for deployment resources.
+  This parameter and `vars_uri` are alternative.
+
+* `template_uri` - (Optional, String) Specifies the OBS address where the HCL/JSON template archive (**.zip** file,
+  which contains all resource **.tf.json** script files to be deployed) or **.tf.json** file is located, which describes
+  the target status of the deployment resources.
+
+* `vars_uri` - (Optional, String) Specifies the OBS address where the variable (**.tfvars**) file corresponding to the
+  HCL/JSON template is located, which describes the target status of the deployment resources.
+
+* `enable_auto_rollback` - (Optional, Bool, ForceNew) Specifies whether to enable automatic rollback.  
+  After enabled, the stack will automatically rollback to the last stable state after the deployment fails.
+  The default value is **true**.
+  Change this parameter will create a new resource.
```

Comment:
```suggestion
  A new resource will be created by value reversion.
  The new resource will be recreated by value reversion.
  （重建？还是新建）
```

Reply:
The current description is the general description for provider documentation.